featureflags/graph/graph.py

```diff
@@ -716,14 +692,34 @@ async def delete_flag(ctx, options):
 MUTATION_GRAPH = apply(MUTATION_GRAPH, [AsyncGraphMetrics("mutation")])
 
 
-@metrics.wrap(graph_pull_time.time())
-@metrics.wrap(graph_pull_errors.count_exceptions())
-async def exec_graph(engine, query, *, sa, session):
-    return await engine.execute(
+@wrap_metric(GRAPH_PULL_TIME_HISTOGRAM.time())
+@wrap_metric(GRAPH_PULL_ERRORS_COUNTER.count_exceptions())
+async def exec_graph(
+    graph_engine: Engine,
+    query: Node,
+    db_engine: aiopg.sa.Engine,
+    session: contextvars.ContextVar[UserSession],
```

Comment:
it is better to pass user_session directly to exec_graph because passing contextvars is not a good patterm

Reply:
done